### ISSUE TITLE
Get grenade's velocity when it just be thrown out.

### ISSUE THREAD

**Issue by hx-w** [markus-wa/demoinfocs-golang#281]
**Higher goal**
I want to get every utility/grenade's velocity when it just be thrown out. I can't find velocity in `GrenadeProjectile`.

**Describe the solution you'd like**
It will be good if there is a `Velocity` key in `GrenadeProjectile` to describe grenade's initial velocity.

**Describe alternatives you've considered**
None.

**Additional context**
None.

**Comment by markus-wa** [comment]
:wave: @hx-w - thanks for the suggestion

I've added it here #286 - could you have a look if that works for you?

Also, if you are interested in the speed rather than velocity, you can do:

```go
var origin r3.Vector
origin.Distance(projectile.Velocity())
```

**Comment by hx-w** [comment]
The  PR is exactly what I want! 👍 
I want to parse player's utility throwing actions in demo and replay those actions in csgo server by sourcemod. This repo helps me A LOT !

**Comment by markus-wa** [comment]
Cool, glad you like the project! I've just released v2.8.0 with this change (and some other stuff).

Your idea sounds great as well, let me know when/if you've got something cool to show off :smile:
